security/aporeto/docs/README.md

```diff
@@ -0,0 +1,337 @@
+  metadata:
+    name: external-ingress
+  spec:
+    description: |
+      Allow the frontend (web) to receive connections from the Internet.
+    source:
+      - - ext:network=any
+    destination:
+      - - app=myapp
+      - - role=web
+      - - env=production
+```
+
+**🤓 ProTip**
+
+* You can define your own external networks using an `ExternalNetwork` described later in this document.
+
+### Namespace to External Network 
+
+This sample policy is used to allow pods to communicate with an external network; typically use this type of policy to talk to the Interweb 🧐. Create a policy similar to this one for each system that needs to **create** connections to the Internet or other parts of the BCGov (SPANBC) network.
+
+The sample below allows the API pod(s) to open connections to any system on the Internet.
+
+```yaml
+- kind: NetworkSecurityPolicy
+  apiVersion: secops.pathfinder.gov.bc.ca/v1alpha1
+  metadata:
+    name: internal-egress
+  spec:
+    description: |
+      Allow the backend (API) to open connections to the
+      Internet.
+    source:
+      - - app=myapp
+      - - role=api
+      - - env=production
+    destination:
+      - - ext:network=any
+```
+
+### Namespace to Namespace
+
+This sample policy is used to allow pods to communicate accross namespaces; typically use this type of policy to talk to other applications hosted on the OCP. Create a policy similar to this one for each system that needs to **create** connections to another OCP namespace. The owner of the `destination` namespace will need a similar policy to allow incoming connections from the `source` namespace. 
+
+The sample below allows the API pod(s) to open connections to a specific pod(s) in a different namespace.
+
+```yaml
+- kind: NetworkSecurityPolicy
+  apiVersion: secops.pathfinder.gov.bc.ca/v1alpha1
+  metadata:
+    name: ns2ns-comms
+  spec:
+    description: |
+      Allow the backend (API) to open connections to backend (API) pod(s)
+      in the handy-dandy-prod namespace.
+    source:
+      - - app=myapp
+      - - role=api
+      - - env=production
+    destination:
+      - - app=theirapp
+      - - role=api
+      - - env=production
+      - - $namespace=handy-dandy-prod
+```
+
+**🤓 ProTip**
+
+* Use enough labels to uniquely identify the target system. Its better to not solely rely on generic tags like `env=production` or `app=theirapp`.
+
+## Usage
+
+Creating a Zero Trust security model is relatively easy; here's how you'll do it: Create your policy based on the information above; then add it to your OCP deployment manifest; and finally, deploy your application. Lets go over them in more details:
+
+**Step 1 - Add Labels***
+
+Open your deployment manifest and add labels to your `DeploymentConfig` sections to uniquely identify each one:
+
+![Add Labels](add_labels.gif)
+
+In this illustration I have a minio deployment that I add the label `role: objstore` to and an API deployment I add `role: api` to.
+
+**Step 2 - Add Policy**
```

Comment:
Added a comment that if an app already has a base custom policy set up, they would have to remove it first, before adding the new policy.

Reply:
Added.